assets/js/translator.js

```diff
@@ -614,7 +618,9 @@ function translateDoc() {
     }
 }
 
-function detectLanguage() {
+function identify(detect=false) {
```

Comment:
Please don't use default parameters: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/default_parameters#Browser_compatibility. Hound is complaining because ES6 isn't enabled. `detect` falsey by default if you don't pass in a value so this should be a very simple fix.